docs/pages/core-concepts.mdx

```diff
@@ -0,0 +1,219 @@
+Proxy Service. In a demo environment, you can run these two services from a
+single **Teleport Process**.
+
+## Teleport Control Plane
+
+The Control Plane consists of the **Auth Service** and **Proxy Service**, which
+govern cluster-wide access. Other **Teleport Services**, which ensure access to
+specific resources, are not considered part of the Teleport Control Plane.
+
+## Teleport Enterprise
+
+**Teleport Enterprise** is a paid plan that includes additional features for the
+**Teleport Control Plane** and **Teleport Services**, plus a support agreement
+with Teleport.
+
+Read more in our [Teleport Enterprise
+guide](./deploy-a-cluster/teleport-enterprise/introduction.mdx).
+
+## Teleport Process
+
+A single running `teleport` binary is called a **Teleport Process**. A Teleport
+Process can run one or more **Teleport Services**, depending on the user's
+configuration. 
+
+Read about all subcommands of `teleport` in our [CLI
+Reference](./reference/cli.mdx#teleport).
+
+## Teleport Proxy Service
+
+The **Teleport Proxy Service** allows for secure access to resources in a
+**Teleport Cluster** from the public internet without the need for a VPN. 
+
+It establishes reverse tunnels to **Teleport Services**, including the
+**Teleport Auth Service**, which can run in private networks. This means that,
+in the Proxy Service's minimal configuration, you can expose only port `443` to
+the internet and run the rest of your infrastructure in private networks.
+
+<Figure
+  align="center"
+  bordered="false"
+  caption="Proxy service"
+>
+
+![Proxy service](../img/architecture/proxy.png)
+
+</Figure>
+
+You can also configure clients to bypass Proxy Service instances and connect to
+resources with Teleport-issued certificates directly. 
+
+Read our guide to [how the Teleport Proxy Service
+works](./architecture/proxy.mdx).
+
+## Teleport Service
+
+A **Teleport Service** manages access to resources in your infrastructure, such
+as Kubernetes clusters, Windows desktops, internal web applications, and
+databases. All Teleport Services can run from a single **Teleport Process**, as
+can the **Auth Service** and **Proxy Service**:
+
+|Service|Traffic it Proxies|
+|---|---|
+|[Teleport Database Service](./database-access/introduction.mdx)|TCP traffic in the native protocols of popular databases, including PostgreSQL and MySQL.|
+|[Teleport Application Service](./application-access/introduction.mdx)|HTTP and TCP traffic to user-configured endpoints, e.g., internal web applications or the AWS Console.|
+|[Teleport Desktop Service](./desktop-access/introduction.mdx)|Remote Desktop Protocol traffic to Windows desktops.|
+|[Teleport Kubernetes Service](./kubernetes-access/introduction.mdx)|HTTP traffic to the Kubernetes API server.|
+|[Teleport Node Service](./server-access/introduction.mdx)|Remote shell command execution via an SSH server implementation. Machines you can access via this service are called **Teleport Nodes**.|
```

Comment:
Let's call this SSH service, since that's how it's referred to when configuring teleport.